index.bs

```diff
@@ -918,16 +918,26 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 : <dfn>Client-side-resident Public Key Credential Source</dfn>
 : <dfn>Resident Credential</dfn>
-:: A [=Client-side-resident Public Key Credential Source=], or [=Resident Credential=] for short, is a [=public key credential
-    source=] whose [=credential private key=] is stored in the [=authenticator=], [=client=] or [=client device=]. Such
-    [=client-side=] storage requires a [=resident credential capable=] [=authenticator=] and has the property that the [=authenticator=]
-    is able to select the [=credential private key=] given
-    only an [=RP ID=], possibly with user assistance (e.g., by providing the user a pick list of [=public key credential|credentials=] [=scoped=] to the [=RP ID=]).
-    By definition, the [=credential private key=] is always exclusively controlled by the [=authenticator=]. In the case of a
-    [=resident credential=], the [=authenticator=] might offload storage of wrapped key material to the
-    [=client device=], but the [=client device=] is not expected to offload the key storage to remote entities (e.g., [=[WRP]=] Server).
+:: A [=Client-side-resident Public Key Credential Source=], or [=Resident Credential=] for short,
+    is a [=public key credential source=] that is discoverable and usable in [=authentication ceremonies=]
+    without providing [=credential ID=]s,
+    i.e., calling {{CredentialsContainer/get()|navigator.credentials.get()}}
+    with an [=list/is empty|empty=] {{PublicKeyCredentialRequestOptions/allowCredentials}} argument.
+
+    As a consequence, a [=resident credential capable=] [=authenticator=] can generate an [=assertion signature=]
+    for a [=resident credential=] given only an [=RP ID=],
```

Comment:
s/ for / using / ?

Reply:
Hm, I see what you mean but I somehow feel like that muddies what I want to express here. I can't quite pinpoint what it is, but I think it has to do with how the "credential" in the challenger's (RP's) point of view is something the challenger has, and the signature is something to match it. Like the "credential" (public key) being a lock and the signature being a key _for_ it (even if the "key" is technically created _using_ the credential source).

Reply:
Yeah, I spose you're looking at it sorta differently than I do.  it's NBD (no big deal).